taipy/gui/viselements.json

```diff
@@ -1735,7 +1735,40 @@
                     }
                 ]
             }
-        ]
+        ],
+        [
+            "alert",  
+            {
+                "inherits": ["shared"],
+                "properties": [
+                    {
+                        "name": "severity",
+                        "type": "dynamic(str)",
+                        "default_value": "\"error\"",
+                        "doc": "Defines the severity level of the notification. Valid values: \"error\", \"warning\", \"info\", \"success\"."
+                    },
+                    {
+                        "name": "message",
+                        "default_property": true,
+                        "type": "dynamic(str)",
+                        "default_value": "\"\"",
+                        "doc": "The message displayed in the notification. Can be a dynamic string."
+                    },
+                    {
+                        "name": "variant",
+                        "type": "str",
+                        "default_value": "\"filled\"",
+                        "doc": "Defines the variant of the notification. Valid values: \"filled\", \"outlined\", \"standard\"."
+                    },
+                    {
+                        "name": "defaultMessage",
```

Comment:
You don't want this.
'defaultMessage' was not meant to be an element property, but a property for the React component: it is set to the initial value of the 'message' React property, when no binding has yet initialized for for display.